code/datums/skills/_skill.dm

```diff
@@ -1,11 +1,22 @@
 /datum/skill
-	var/name = "Skill"
+	var/name = "Skilling"
+	var/jobname = "Skiller"
```

Comment:
rename this to title or something jobname sounds odd and I had to look at implementation, also, autodoc all of this stuff please